packages/@uppy/url/src/Url.jsx

```diff
@@ -88,6 +88,9 @@ export default class Url extends UIPlugin {
       companionHeaders: this.opts.companionHeaders,
       companionCookiesRule: this.opts.companionCookiesRule,
     })
+
+    this.requestClientId = 'url';
```

Comment:
What about reusing `this.id?` And why are we only setting this for the url plugin, not all other providers?

Reply:
it's a bit hard to see between all the whitespace changes, but i also did the same in View.js (used by providers). as for reusing this.id, it feels a bit error prone, if someone customizes their `id`, then it could blow up

Reply:
But they could also customize `requestClientId`, no? Should we use `.constructor.name` instead? That seems unlikely to change

Reply:
ok have now  refactored to use constructor.name, but in View.js we have to use this.provider.provider because we cannot get constructor.name